pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.573</version>
+    <version>1.599</version>
```

Comment:
I am not directly using `SimpleBuildWrapper` so I can remove the unused import has been removed. `MaskPasswordsBuildWrapper` is a directly extends `SimpleBuildWrapper`. The build fails with build version `1.573`.